qml/pages/CameraUI.qml

```diff
@@ -608,4 +629,26 @@ Page {
 
         return siz.width + "x" + siz.height;
     }
+
+    RoundButton {
+        id: btnGallery
+
+        visible: false
+        enabled: false
+
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: 50
```

Comment:
Hardcoding margins with magic number is not a proper way, we should use Silica margin values when possible.

Reply:
Not sure how to handle this one...